### ISSUE TITLE
Is there a way to change the size of the window?

### ISSUE THREAD

**Issue by EinTheMidle** [Splode/pomotroid#149]
It's too big for the wat I set my screen, if there's a way to change it it would be very useful

**Comment by faztp12** [comment]
This would be really helpful. Maybe a mini-pomotroid view ?

**Comment by LinaraKhis** [comment]
Yes, It would be awesome to have a mini view for using with on top mode. Something about 1/4 of it size now would be perfect.

P.S. I'm in love with this timer! I think it is the best pomodoro app what I seen.  Thank you!

**Comment by Splode** [comment]
[v1.0.0](https://github.com/Splode/pomotroid/releases/tag/v1.0.0) adds support for resizing, and a mini mode.

**Comment by EinTheMidle** [comment]
Damn, if adding windows resizing for an app took this long, now I can understand why it take that long to ralease GTA VI hahaha 
P.D. thanks mate
